### PR TITLE
new p content

### PR DESCRIPTION
**Pull request by EmilyRead88**
added beautiful text in paragraph.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,5 +8,8 @@
 </head>
 <body>
      <h1>What a nice kitty</h1>
+
+     <p>THIS IS GOOD CONTENT THIS IS GOOD CONTENT THIS IS GOOD CONTENT... SUCH GOOD CONTENT</p>
```
Comment (sophie1620) [inline index.html:12]: This is such AWESOME content!  Love how clean your code is!

**Review by sophie1620 — CHANGES_REQUESTED** [review]
Looking GRRRREAT! (No changes need to be made, but I'm going to request change to follow along in class)

**Comment by EmilyRead88** [comment]
made that content extra awesome

File: index.html (hunk 1 above)
Comment (EmilyRead88) [reply index.html]: resolved

**Review by sophie1620 — APPROVED** [review]
EVERYTHING IS AWESOME!! Go ahead and merge.

**Pull request merged**